entrypoints/services/FiTransactionService.ts

```
@@ -0,0 +1,197 @@
+    private static readonly SCHEMA_VERSION = 1;
+
+    /**
+     * Create a new FI transaction
+     */
+    static async create(
+        transaction: Omit<FiTransaction, "id" | "schemaVersion">
+    ): Promise<FiTransaction> {
+        try {
+            const newTransaction: FiTransaction = {
+                ...transaction,
+                schemaVersion: this.SCHEMA_VERSION,
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)